src/apps/lwaftr/lwaftr.lua

```diff
@@ -243,7 +200,8 @@ local function binding_lookup_ipv4(lwstate, ipv4_ip, port)
       print(lwdebug.format_ipv4(ipv4_ip), 'port: ', port, string.format("%x", port))
       lwdebug.pp(lwstate.binding_table)
    end
-   local psid = lwstate.psid_info_map:lookup_psid(ipv4_ip, port)
+   local host_endian_ipv4 = C.htonl(ipv4_ip)
```

Comment:
Is this correct? IPv4 addresses from packets are in network byte order, thus `ipv4_ip` is in network byte order. Shouldn't be `local host_endian_ipv4 = C.ntohl(ipv4_ip)` instead? I believe it works because in the end `hton` and `ntoh` what they do is to swipe byte order.